experiments/AMIP/modular/components/atmosphere/climaatmos_init.jl

```diff
@@ -113,10 +114,24 @@ function compute_atmos_turbulent_fluxes!(atmos_sim::ClimaAtmosSimulation, csf)
 
     p = atmos_sim.integrator.p
 
+    thermo_params = get_thermo_params(atmos_sim)
+    ts_int = get_field(atmos_sim, Val(:thermo_state_int))
+
+    # surface density is needed for q_sat and requires atmos and sfc states, so it is calculated and saved in the coupler
+    parent(csf.ρ_sfc) .=
+        parent(extrapolate_ρ_to_sfc.(Ref(thermo_params), ts_int, swap_space!(ones(axes(ts_int.ρ)), csf.T_S)))
+
+    # surface specific q_vap is calculated as a bulk quantity here, assuming a saturated surface.
+    # TODO - use land's q_sfc, but ClimaLSM need to be modified to ingest model type
+    # NB: q_sfc (q_sfc is actually saturated!) is a passive variable in Land and nonexistent in the slabs, so this is ok for testing,
```

Comment:
I don't quite understand this series of comments. Here, are you saying it's okay to specify `q_vap` in the coupler since it's not involved in component model calculations?